### PR TITLE
[TH-351] Add campaign step 3 page UI

### PR DESCRIPTION
**Pull request by ssk7808**
# Description
- step 3 : bridge UI 만 구현하였습니다. 

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can
reproduce. Please also list any relevant details for your test configuration

- [ ] Test A : 아무 숫자나 넣고 Bridge 클릭 시 성공화면 이동

## Screenshots (if appropriate):

https://github.com/TeamHeimdallr/moai-web/assets/29878878/a7998dbc-3d83-43b7-b788-84ba7c664cf1

### CODE REVIEW

File: src/pages/campaign/components/input-number/index.tsx
Comment (Imchaemin) [inline src/pages/campaign/components/input-number/index.tsx:1]: 어떤부분이 달라서 새로 만드셨나요??

File: src/pages/campaign/components/list/index.tsx
Comment (Imchaemin) [inline src/pages/campaign/components/list/index.tsx:1]: 여기도 기존 컴포넌트를 업데이트 하는방식으로는 어려울까요?

Comment (ssk7808) [reply src/pages/campaign/components/list/index.tsx:1]: 기존 컴포넌트 업데이트하는걸로 다시 하겠습니당

File: src/pages/campaign/components/input-number/index.tsx
Comment (ssk7808) [reply src/pages/campaign/components/input-number/index.tsx:1]: 요것도 다시 해볼게요

File: src/pages/campaign/components/list/index.tsx
Comment (ssk7808) [reply src/pages/campaign/components/list/index.tsx:1]: [fix: use global component - input, list](https://github.com/TeamHeimdallr/moai-web/pull/233/commits/219d8e43403a743c7e152801c7efde70052f3bbb)

File: src/pages/campaign/components/input-number/index.tsx
Comment (ssk7808) [reply src/pages/campaign/components/input-number/index.tsx:1]: [fix: use global component - input, list](https://github.com/TeamHeimdallr/moai-web/pull/233/commits/219d8e43403a743c7e152801c7efde70052f3bbb)

**Review by Imchaemin — APPROVED** [review]

**Pull request merged**